src/main/webapp/resources/js/components/project-members/ProjectMembersTable.jsx

```diff
@@ -69,7 +69,7 @@ export function ProjectMembersTable({ projectId }) {
     },
   ];
 
-  if (project.canManage) {
+  if (project.canManage || project.canManageRemote) {
```

Comment:
you can test this, but i'm pretty sure this can just be `canManageRemote`.  If `canManageRemote` is true, `canManage` will also be true.

Reply:
Yup that worked. Thanks. Updated in 7f7480049c4741f4b066167fbc1dbd994df5ccb8